docs/rest-apis/algod/v2.md

```diff
@@ -890,7 +985,7 @@ POST /v2/teal/dryrun
 
 
 **Description**
-Executes TEAL program(s) in context and returns debugging information about the execution. This endpoint is only enabled when a node's configuration file sets EnableDeveloperAPI to true.
+Executes TEAL program(s) in context and returns debugging information about the execution. This endpoint is only enabled when a node's configureation file sets EnableDeveloperAPI to true.
```

Comment:
here too?

Reply:
TODO: fix this in go-algorand source

Reply:
Merged @nullun's https://github.com/algorand/go-algorand/pull/3741